packages/terra-menu/src/MenuDivider.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Menu.scss';
+
+const cx = classNames.bind(styles);
+
+
```

Comment:
Nit: extra line. Also cx('divider') will result in the same style-binded classname generation.